app/code/community/Qenta/CheckoutPage/Model/System/Config/InvoiceProviders.php

```diff
@@ -30,14 +30,14 @@
  * Please do not use the plugin if you do not agree to these terms of use!
  */
 
-class Wirecard_CheckoutPage_Model_System_Config_InvoiceProviders
+class Qenta_CheckoutPage_Model_System_Config_InvoiceProviders
 {
     public function toOptionArray()
     {
         $themes = array(
             array('value' => 'payolution', 'label' => 'payolution'),
             array('value' => 'ratepay', 'label' => 'RatePay'),
-            array('value' => 'ratepay', 'label' => 'Wirecard'),
+            array('value' => 'ratepay', 'label' => 'Qenta'),
```

Comment:
This label cannot be changed to Qenta as Qenta is not an invoice provider. Please rather remove this line

Reply:
Resolved